ACE/ace/Log_Msg_Android_Logcat.h

```diff
@@ -35,7 +35,6 @@ ACE_BEGIN_VERSIONED_NAMESPACE_DECL
 class ACE_Export ACE_Log_Msg_Android_Logcat : public ACE_Log_Msg_Backend
 {
 public:
-  ACE_Log_Msg_Android_Logcat () = default;
```

Comment:
Why remove it, when marked with default it is clear that the default constructor is there but is just the default compiler generated

Reply:
No matter whether to define the default ctor, the compiler will generate it.

Reply:
Just let it stay, makes clear the compiler is responsible for the default

Reply:
OK